e2e/cephfs.go

```diff
@@ -351,12 +359,20 @@ var _ = Describe("cephfs", func() {
 						e2elog.Failf("failed to create application: %v", err)
 					}
 					validateSubvolumeCount(f, 1, fileSystemName, subvolumegroup)
+					pool, err := getCephFSPoolName(f, fileSystemName)
+					if err == nil {
```

Comment:
Hope the check will be inverted like below

```suggestion
pool, err := getCephFSPoolName(f, fileSystemName)
if err != nil {
e2elog.Failf(err)
}
```